MEIOUandTaxes1/src/events/DG-02-Protestant.txt

```diff
@@ -1,3 +1,10172 @@
+						}
+						set_province_flag = mn_unrest_imp_5
+					}
+				}
+			}
+			10 = {
+				if = {
+					limit = {
+						MN_CheckProp = { religion = 2 type = TotP prop = 60 }
+					}
+					add_or_extend_province_modifier_hidden_effect = { NAME = MN_imp_U.02 DURATION = 3650 }
+					if = {
+						limit = {
+							owner = { has_country_flag = pro_unrest_info }
+						}
+						set_province_flag = mn_unrest_imp_10
+					}
+				}
+			}
+			5 = {
+				if = {
+					limit = {
+						MN_CheckProp = { religion = 2 type = TotP prop = 80 }
+					}
+					add_or_extend_province_modifier_hidden_effect = { NAME = MN_max_U.16 DURATION = 365 }
+					if = {
+						limit = {
+							owner = { has_country_flag = pro_unrest_info }
+						}
+						set_province_flag = mn_unrest_max_1
+					}
+				}
+			}
+			10 = {
+				if = {
+					limit = {
+						MN_CheckProp = { religion = 2 type = TotP prop = 80 }
+					}
+					add_or_extend_province_modifier_hidden_effect = { NAME = MN_max_U.16 DURATION = 1825 }
+					if = {
+						limit = {
+							owner = { has_country_flag = pro_unrest_info }
+						}
+						set_province_flag = mn_unrest_max_5
+					}
+				}
+			}
+			5 = {
+				if = {
+					limit = {
+						MN_CheckProp = { religion = 2 type = TotP prop = 80 }
+					}
+					add_or_extend_province_modifier_hidden_effect = { NAME = MN_max_U.16 DURATION = 3650 }
+					if = {
+						limit = {
+							owner = { has_country_flag = pro_unrest_info }
+						}
+						set_province_flag = mn_unrest_max_10
+					}
+				}
+			}
+		}
+		if = {
+			limit = {
+				owner = { has_country_flag = pro_unrest_info }
+			}
+			province_event = { id = DG_Protestant.312 }
+		}
+	}
+}
+
+province_event = {
+	id = DG_Protestant.312
+	title = "DG_Protestant.311.title"
+	desc = "DG_Protestant.311.desc"
+	picture = all_church_state
+	
+	is_triggered_only = yes
+	
+	option = {
+		name = "OPT.OHNO"
+		trigger_switch = {
+			on_trigger = has_province_flag
+
+			mn_unrest_min_1 =	{ custom_tooltip = mn_unrest_min_1_tt }
+			mn_unrest_min_5 =	{ custom_tooltip = mn_unrest_min_2_tt }
+			mn_unrest_min_10 =	{ custom_tooltip = mn_unrest_min_3_tt }
+			mn_unrest_low_1 =	{ custom_tooltip = mn_unrest_low_1_tt }
+			mn_unrest_low_5 =	{ custom_tooltip = mn_unrest_low_2_tt }
+			mn_unrest_low_10 =	{ custom_tooltip = mn_unrest_low_3_tt } 
+			mn_unrest_mid_1 =	{ custom_tooltip = mn_unrest_mid_1_tt }
+			mn_unrest_mid_5 =	{ custom_tooltip = mn_unrest_mid_2_tt }
+			mn_unrest_mid_10 =	{ custom_tooltip = mn_unrest_mid_3_tt }
+			mn_unrest_imp_1 =	{ custom_tooltip = mn_unrest_imp_1_tt }
+			mn_unrest_imp_5 =	{ custom_tooltip = mn_unrest_imp_2_tt }
+			mn_unrest_imp_10 =	{ custom_tooltip = mn_unrest_imp_3_tt } 
+			mn_unrest_max_1 =	{ custom_tooltip = mn_unrest_max_1_tt }
+			mn_unrest_max_5 =	{ custom_tooltip = mn_unrest_max_2_tt }
+			mn_unrest_max_10 =	{ custom_tooltip = mn_unrest_max_3_tt } 
+		}
+	}
+	after = {
+		hidden_effect = {
+			UnrestNofificationCleanup = yes
+		}
+	}
+}
+=======
```

Comment:
This file is broken due to merge problem. Need to be fixed before merging the pull request.